theme/templates/base.html

```diff
@@ -109,5 +109,9 @@
                 gtag('config', '{{ GOOGLE_ANALYTICS }}');
             </script>
         {% endif %}
+        {% if ADDTHIS_PUBID %}
```

Comment:
This block should be in the same template where the buttons are shown. No need to load extra JavaScript if we're not going to use it.

In this case you may not take the request "place this script at the bottom of the page" literally.